src/dialects/maria/index.js

```diff
@@ -21,6 +21,8 @@ assign(Client_MariaSQL.prototype, {
 
   driverName: 'mariasql',
 
+  SqlString: require('../../query/string'),
```

Comment:
This should not be needed since `src/client.js` base class already sets it to `src/query/string.js`